### PR TITLE
Fix problem with multi  frsky fallback

### PR DESCRIPTION
**Pull request by schwabe**

### CODE REVIEW

File: radio/src/telemetry/multi.cpp [hunk 1]
```diff
@@ -416,14 +422,35 @@ void processMultiTelemetryData(const uint8_t data)
       processMultiTelemetryByte(data);
       break;
 
+    case MultiStatusOrFrskyData:
+      // Check len byte if it makes sense for multi
+      if (data >= 5 && data <= 10) {
+        multiTelemetryBufferState = ReceivingMultiStatus;
+        telemetryRxBufferCount = 0;
+      } else
```
Comment (bsongis) [inline radio/src/telemetry/multi.cpp]: code style!

**Comment by bsongis** [comment]
I am rather reticent for 2.2.1, what is the issue?

**Comment by schwabe** [comment]
When multi code is compiled with multi status (officially more for erksy9x and er9x than opentx), the current code switches to frsky when it sees a 0x7e and never switches back to multi telemetry. 

It is not important for 2.2.1 as users can always also compile their modules with MULTI_TELEMETRY which works fine.

**Comment by bsongis** [comment]
ok I will merge it just after 2.2.1 is released then

**Comment by bsongis** [comment]
Time to merge this one as 2.2.1 is out

**Pull request merged**